server/etcdmain/config.go

```diff
@@ -115,7 +115,11 @@ func newConfig() *config {
 	)
 	fs.Var(
 		flags.NewUniqueURLsWithExceptions(embed.DefaultListenClientURLs, ""), "listen-client-urls",
-		"List of URLs to listen on for client traffic.",
+		"List of URLs to listen on for client grpc traffic. Http services are available as long --listen-client-http-urls is not specified.",
+	)
+	fs.Var(
+		flags.NewUniqueURLsWithExceptions("", ""), "listen-client-http-urls",
```

Comment:
Do we need to add the corresponding `--advertise-client-http-urls`?

Reply:
No, as etcd client and members only communicate via grpc to client port.

Reply:
I assume a scenario when someone lists members, takes 'advertise-client-url` from the output and uses it for metrics gathering / http communication. That is fair to say that the advertised service in case of split, points on the grpc service.